android/src/main/java/com/razorpay/rn/RazorpayModule.java

```diff
@@ -16,6 +16,9 @@
 import com.facebook.react.bridge.Arguments;
 import com.facebook.react.modules.core.DeviceEventManagerModule;
 import com.razorpay.CheckoutActivity;
+import com.razorpay.PaymentData;
```

Comment:
Have we decided explicitly that we are exposing this? Because then, it needs to be consistent across all public repos (cordova, sample app, etc.)